packages/utils/src/lib/report-to-md.ts

```diff
@@ -95,38 +87,17 @@ function reportToCategoriesSection(report: ScoredReport): string {
       category.score,
     )} Score:  ${style(formatReportScore(category.score))}`;
     const categoryDocs = getDocsAndDescription(category);
-
-    const auditsAndGroups = category.refs.reduce(
-      (
-        acc: {
-          audits: WeighedAuditReport[];
-          groups: EnrichedScoredAuditGroupWithAudits[];
-        },
-        ref: CategoryRef,
-      ) => ({
-        ...acc,
-        ...(ref.type === 'group'
-          ? {
-              groups: [
-                ...acc.groups,
-                getGroupWithAudits(ref.slug, ref.plugin, plugins),
-              ],
-            }
-          : {
-              audits: [...acc.audits, getAuditByRef(ref, plugins)],
-            }),
-      }),
-      { groups: [], audits: [] },
-    );
-
-    const audits = auditsAndGroups.audits
-      .sort(sortCategoryAudits)
-      .map(audit => auditItemToCategorySection(audit, plugins))
-      .join(NEW_LINE);
-
-    const groups = auditsAndGroups.groups
-      .map(group => groupItemToCategorySection(group, plugins))
-      .join('');
+    const categoryItems = category.refs.reduce((acc, ref) => {
```

Comment:
The naming `categoryItems` does not tell me what is is... Is it scored refs? Is it categories?

Reply:
It's the markdown string, containing categories, which include their audits and groups of audits. I already have `categories` variable here, so I'll better rename it to `categoryMDItems`, I hope it would be clearer.